api/ipfsproxy/config.go

```diff
@@ -317,7 +317,15 @@ func (cfg *Config) toJSONConfig() (jcfg *jsonConfig, err error) {
 	return
 }
 
-func (cfg *Config) String() string {
-	jcfg, _ := cfg.toJSONConfig()
-	return config.String(*jcfg, nil)
+func (cfg *Config) String() (string, error) {
+	jcfg, err := cfg.toJSONConfig()
+	if err != nil {
+		return "", err
+	}
+
+	bytes, err := config.DefaultJSONMarshalWithoutHiddenFields(*jcfg)
```

Comment:
Rather than de-referencing everytime, it is best to fix the function to de-reference automatically when the value passed in is a pointer.